tasks/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 # Variable configuration.
 - include: variables.yml
+- include: cloudhost.yml
+  when: cloudhost
```

Comment:
when would this not be true?

Reply:
I just added it in in case we wanted to use the default role. Was not sure if it was used for anything else.  I can remove it and always include if that is not the case.